### ISSUE TITLE
VRCX will cause huge bug that i cant open any application depend on steamvr

### ISSUE THREAD

**Issue by 1286365975** [vrcx-team/VRCX#1058]
### VRCX 20241203 will cause i cant open any application depend on steamvr!!!!!!!!!!!!!!!!!!!!!
IN vrchat log,it will casuse

```
2024.12.28 00:29:34 Error      -  [OpenVR] Could not initialize OpenVR. Error code: IPC_NamespaceUnavailable


2024.12.28 00:29:34 Error      -  Initializing XR Failed. Check Editor or Player log for details.
```

and sometimes,my steam vr will report
```
SteamVR initialization failed!
SteamVR failed initialization with error code VRinitError IPC NamespaceUnavailable: "shared IPc Namespace Unavailable (310)
Please verify SteamVR is properly installed and try again.
```
It took me 5 hours to find out that VRCX cause this
Fix it as soon as possible

Follow is steamvr system report
[SteamVR-2024-12-28-AM_02_01_47.txt](https://github.com/user-attachments/files/18262610/SteamVR-2024-12-28-AM_02_01_47.txt)

**Comment by Natsumi-sama** [comment]
Could you give this option a try, `Settings -> General -> Disable VR Overlay GPU Acceleration`

**Comment by Lunanite** [comment]
> Could you give this option a try, `Settings -> General -> Disable VR Overlay GPU Acceleration`

I was also experiencing this bug, disabling either or both of the GPU accelerations does not fix the bug.

**Comment by Natsumi-sama** [comment]
Restarted VRCX after changing the options? also tried disabling VR overlay all together?

**Comment by Lunanite** [comment]
> Restarted VRCX after changing the options? also tried disabling VR overlay all together?

Disabling the VR overlay does stop the bug.

**Comment by YanYiZA** [comment]
Yes,Disable VR Overlay GPU Acceleration doesnt work.but disabling vr overlay can stop it.You'd better try to find out why,because i dont even know that vrcx cause this bug and its difficult to find out it

**Comment by Natsumi-sama** [comment]
I'd like to see the VRCX logs
`%AppData%\VRCX\logs`
`C:\Program Files (x86)\Steam\logs\vrclient_VRCX.txt`
